lib/Compiler.cpp

```diff
@@ -626,6 +627,7 @@ int RunPassPipeline(llvm::Module &M, llvm::raw_svector_ostream *binaryStream) {
     // This pass mucks with types to point where you shouldn't rely on
     // DataLayout anymore so leave this right before SPIR-V generation.
     pm.addPass(clspv::UBOTypeTransformPass());
+    pm.addPass(clspv::SetImageChannelMetadataPass());
```

Comment:
This probably deserves a comment on the dependency with inlining.

Reply:
Done